core/jobs/types/firebase_domain.py

```
@@ -0,0 +1,121 @@
+                f'{user_auth_details_model.id=!r} with '
+                f'deleted={user_auth_details_model.deleted} must match '
+                f'{user_settings_model.id=!r} with '
+                f'deleted={user_settings_model.deleted}'
+            )
+        if user_auth_details_model.parent_user_id is not None:
+            return None
+        if user_auth_details_model.firebase_auth_id is None:
+            raise ValueError(
+                f'{user_auth_details_model.id=!r} must have non-empty'
+                ' firebase_auth_id'
+            )
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)